package.json

```diff
@@ -61,13 +53,10 @@
     "lint": "eslint --ext .js,.jsx src",
     "build": "npm run build:js && npm run build:scss",
     "build:js": "webpack -d",
-    "postbuild:js": "uglifyjs build/js/lyra.js -cm > build/js/lyra.min.js",
     "build:scss": "node-sass --include-path src/scss src/scss/main.scss build/css/main.css",
-    "postbuild:scss": "cleancss -o build/css/main.min.css build/css/main.css",
```

Comment:
AHA you are correct. Gon' re-write the HTML to reference `main.css` instead of the minified file, for now.